lib/money-rails/helpers/action_view_extension.rb

```diff
@@ -16,10 +16,12 @@ def humanized_money(value, options={})
         :symbol => false
       }.merge(options)
       options.delete(:symbol) if options[:disambiguate]
-
+      
       if value.is_a?(Money)
+        value = value.to_d.round.to_money if options[:no_cents] == true && value.present?
```

Comment:
The extra `value.present?` check is unnecessary since this branch is encountered only if `value.is_a?(Money)`.